### PR TITLE
Second implementation with composer and finder

### PR DESCRIPTION
**Pull request by Mte90**
It is time for a review!

### CODE REVIEW

File: finder.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+return \StubsGenerator\Finder::create()
+    ->in('source/vendor/wp-cli/wp-cli')
+    ->append(\StubsGenerator\Finder::create()->files()->depth('< 4')->in(['source/vendor/wp-cli/wp-cli/php']))
```
Comment (szepeviktor) [inline finder.php]: these "append"-s are not doing anything

Comment (szepeviktor) [reply finder.php]: Basically we need two things from a command or wp-cli/wp-cli:

- the main command file
- and classes

sometimes the command file is almost empty: https://github.com/wp-cli/core-command/blob/master/core-command.php so only the classes

Comment (szepeviktor) [reply finder.php]: Actually all the `*command.php` files are tiny: WP-CLI must have moved everything into classes.

Comment (Mte90) [reply finder.php]: changed the depth to 2

**Pull request merged**